### ISSUE TITLE
Evaluate hidden/disabled at onCellSelection callback

### ISSUE THREAD

**Issue by phiberjenz** [xmartlabs/Eureka#10]
Maybe I'm doing it wrong, but is it possible to evaluate if a row/section should be hidden based on if another row is selected?

I can get it to work when the value of the other row changes, but not when it is selected.

EDIT:
Is there no possibility to set the hidden status of a row from another function? I'm trying to set one rows hidden status from another rows onCellSelection handler, but it doesn't work. I can hide the CELL, but not the row. Any ideas?

Regards,
Jens

**Comment by mtnbarreto** [comment]
@phiberjenz I have never tried but i think this may work:
I'm not sure what row selection means for you... 

```swift
row.onCellSelection { [weak row2] in cell, row in  // on cell selection is invoked every time didSelectCellAtIndexPath Is Invoked
        row2.evaluateHidden()
}
```

if it doesn't work you can also play with `onCellHighlight` callback that was added to master branch (not available on 1.0.0).

**Comment by phiberjenz** [comment]
Thanks, evaluateHidden() did the trick. Works like a charm.
